script/Run.sc

```diff
@@ -32,27 +32,26 @@ def datetime = {
   new SimpleDateFormat("yyyy/MM/dd HH:mm:ss").format(Calendar.getInstance().getTime()) 
 }
 
-def updatingSubmodules(submodules: List[Path])(f: () => Unit): Unit = {
-  submodules.foreach{ submodule =>
-    runD("git", "checkout", "master")(submodule)
-    runD("git", "remote", "update")(submodule)
-    runD("git", "reset", "--hard", "origin/master")(submodule)
-    runD("git", "pull", "origin", "master")(submodule)
-  }
+def updatingRepositories(contribPath: Path, indexPath: Path)(f: () => Unit): Unit = {
+  // Fetch the last data from the contrib repository
+  runD("git", "checkout", "master")(contribPath)
+  runD("git", "remote", "update")(contribPath)
+  runD("git", "pull", "origin", "master")(contribPath)
 
   // run index
   f()
 
-  // publish the latest data
-  submodules.foreach{ submodule =>
-    runD("git", "add", "-A")(submodule)
-    runD("git", "commit", "-m", '"' + datetime + '"' )(submodule)
-    runD("git", "pull", "origin", "master")(submodule)
-    runD("git", "push", "origin", "master")(submodule)
-  }
+  // Then push the server’s data (live + sonatype notifications) on the index repository
+  runD("git", "add", "-A")(indexPath)
+  runD("git", "commit", "-m", '"' + datetime + '"')(indexPath)
+  runD("git", "push", "origin", "master")(indexPath)
```

Comment:
We commit (and push to the github remote) changes applied to the `index` repository.